site/src/pages/index.js

```diff
@@ -15,6 +15,7 @@ const Home = ({ data, location }) => {
       <Seo
         data={data}
         location={location}
+        title='Home'
```

Comment:
I'm not a big fan of having "Home" or "Index" or anything on the homepage title. I generally think. having just the site title on the homepage is the best approach. This is just an opinion, not a required change.

Reply:
I agree. Seems unnecessary. Removed it.